giga-segy-core/Cargo.toml

```diff
@@ -20,11 +20,12 @@ ibmfloat = "0.1"
 num = "0.4"
 num-derive = "0.3"
 num-traits = "0.2"
-serde = { version = "1.0.40", features = ["derive"], optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
I would prefer keeping "1.0.40" here as long as we don't have verified that the crate also builds with version `1.0.0`.
(It likely won't build as the `1.0.0` version did not have the "derive" feature flag)

Reply:
I have now updated (most) things in accordance with the lib.rs advisory. Since cbindgen 0.26 requires serde 1.0.103, I have updated the minimum serde requirement to 1.0.103 for consistency. Strictly speaking, though, if we're compiling without cbindgen we can use a much older serde version.